### PR TITLE
book: point to reference section, which defines traits

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: src/doc/trpl/traits.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ impl Circle {
 
 [methodsyntax]: method-syntax.html
 
-Traits are similar, except that we define a trait with just the method
+[Traits](../reference.html#traits) are similar,
```
Comment (steveklabnik) [inline src/doc/trpl/traits.md]: We don't generally link to the reference from the book, or at least, not that I'm aware of. So, while I don't think this is wrong, but it is a bit unusual.

Comment (tshepang) [inline src/doc/trpl/traits.md]: Alternatively, how/where would you insert the definition of *trait* in this chapter?

Comment (steveklabnik) [inline src/doc/trpl/traits.md]: This is the traits chapter, so it should explain it.

Comment (tshepang) [inline src/doc/trpl/traits.md]: I mean where in the chapter should it be defined? Maybe before the first paragraph?

Comment (tshepang) [inline src/doc/trpl/traits.md]: New PR: https://github.com/rust-lang/rust/pull/26118

**Pull request closed without merging**